code/Obj/ObjFileParser.cpp

```diff
@@ -66,8 +66,7 @@ ObjFileParser::ObjFileParser()
 , m_pModel( nullptr )
 , m_uiLine( 0 )
 , m_pIO( nullptr )
-, m_progress( nullptr )
-, m_originalObjFileName() {
```

Comment:
Why?

Reply:
because the constructor for those fields it will be called anyway